packages/hooks/README.md

```diff
@@ -29,10 +29,13 @@ function MyApp({ Component, pageProps }) {
 
 The following hooks are available:
 
-- [useWallet](#usewallet)
-- [useContract](#usecontract)
-- [useTransaction](#usetransaction)
-- [useTokenBalance](#usetokenbalance)
+- [@web3-ui/hooks](#web3-uihooks)
+  - [Getting started](#getting-started)
+  - [Hooks](#hooks)
+    - [useWallet](#usewallet)
+    - [useContract](#usecontract)
+    - [useTransaction](#usetransaction)
+    - [useTokenBalance](#usetokenbalance)
```

Comment:
My VSCode extension did magic here, I reverted the changes but it was suggesting that back.

Reply:
Probably a good idea to revert this.

Reply:
Reverted